src/dcurl.c

```diff
@@ -99,6 +117,17 @@ void dcurl_destroy()
 }
 
 
+/**
+ * @brief dcurl execution.
+ *
+ * Retrieve the available hardware from the list and use it to do the
+ * PoW(Proof-of-Work).
+ * @param [in] trytes The trytes for PoW calculation.
+ * @param [in] mwm The minimum weight magnitude.
+ * @param [in] threads The thread number of calculating the PoW. It affects CPU
```

Comment:
Detailed explanation of thread amount may needed (like 0 is actually max amount - 1.)

Reply:
The explanation of threads has been updated.